java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -686,6 +688,13 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         transaction.geopos("key", new String[] {"Place"});
         results.add(Pair.of(GeoPos, buildArgs("key", "Place")));
 
+        transaction.geoadd("key", Map.of("Place2", new GeospatialData(11.0, 20.0)));
```

Comment:
You already tested `geoadd`.
It is a UT, it does not submit a transation to redis.